css/qmwp-rating-faces.css

```diff
@@ -25,7 +25,7 @@
     bottom: 32%;
 }
 
-#sectionSendingMood {
+#sectionSendingRating {
     display: block;
     clear: both;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The property -moz-animation-fill-mode is compatible with -webkit-animation-fill-mode and should be included as well.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1045701984/issues/source?bid=2836188&fileBranchId=2882408#l50)